src/app/forms/components/brakes/brakes.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { FormArray } from '@angular/forms';
+import { DynamicFormService } from '@forms/services/dynamic-form.service';
+import { CustomFormGroup, FormNode, FormNodeOption, FormNodeWidth } from '@forms/services/dynamic-form.types';
+import { TrlBrakes } from '@forms/templates/trl/trl-brakes.template';
+import { TechRecordModel } from '@models/vehicle-tech-record.model';
+import { Subject, debounceTime, takeUntil } from 'rxjs';
+
+@Component({
+  selector: 'app-brakes[vehicleTechRecord]',
+  templateUrl: './brakes.component.html',
+  styleUrls: ['./brakes.component.scss']
+})
+export class BrakesComponent implements OnInit, OnDestroy {
+  @Input() vehicleTechRecord!: TechRecordModel;
+  @Input() isEditing = false;
+  @Output() formChange = new EventEmitter();
+
+  form!: CustomFormGroup;
+  template!: FormNode;
+
+  booleanOptions: Array<FormNodeOption<boolean>> = [{ value: true, label: 'Yes' }, { value: false, label: 'No' }];
+
+  private destroy$ = new Subject<void>();
+
+  constructor(private dfs: DynamicFormService) {}
+
+  ngOnInit(): void {
+    this.template = TrlBrakes;
+
+    this.form = this.dfs.createForm(this.template, this.vehicleTechRecord) as CustomFormGroup;
+
+    this.form.cleanValueChanges
+      .pipe(debounceTime(400), takeUntil(this.destroy$))
+      .subscribe(e => this.formChange.emit(e));
+  }
+
```

Comment:
Does this not need the patch values on change thing too?

Reply:
Yes, it does need it. Thanks. Added.